lang/LangSource/PyrMessage.cpp

```diff
@@ -483,7 +485,7 @@ void prepareArgsForExecute(VMGlobals* g, PyrBlock* block, PyrFrame* callFrame, l
             keywordArgumentsArray->size = keywordArgumentSize;
             SetObject(outCallFrameStack + methNumNormArgs + 1, keywordArgumentsArray);
         } else {
-            SetObject(outCallFrameStack + methNumNormArgs + 1, slotRawObject(&proto->slots[methNumNormArgs + 1]));
+            slotCopy(outCallFrameStack + methNumNormArgs + 1, proto->slots + methNumNormArgs + 1);
```

Comment:
This was the line that was crashing. The default is nil, not an empty array so you can't use SetObject.